### DISCUSSION TITLE
No network devices available in arch linux

### DISCUSSION THREAD

**Discussion by annaxiomm** [utmapp/UTM#4033] [UTM Help (macOS)]
Booted up the VM, tried to connect to the internet using ```iwctl```. No network devices are listed when I use ```device list```, but I've configured the network devices in the VM settings

**Configuration**
* UTM Version: 3.1.5
* OS Version: (Guest) Arch Linux x86_64 2022.05.01 (Host) macOS Monterey 12.0.1
* Intel or Apple Silicon? Silicon

**Comment by peterneutron** [comment]
The `w` in `iwctl` stands for wireless. Did you use usb passthrough with a wireless adapter?

**Comment by peterneutron** [comment]
Do what exactly? If you want basic network access from the VM you need to configure a network device in the Host VM settings. Simplest is the default `Shared Network`. In the Guest VM you run `ip a`, this should list all network devices available. For how to configure basic networking in Arch Linux refer to https://wiki.archlinux.org/title/Network_configuration

**Comment by shlyakpavel** [comment]
https://github.com/utmapp/UTM/issues/3386
https://github.com/utmapp/UTM/issues/3888
https://github.com/utmapp/UTM/issues/4014#issuecomment-1137108988
Discussed many times. I recommend closing this as duplicate.

**Comment by shlyakpavel** [comment]
@[yescyyt](https://github.com/yescyyt) I'm sure your kernel version doesn't match the modules installed in your system. Check `name -a`
The last link in my previous message is a quick way to fix it.
